pkg/handlers/v1/config.go

```diff
@@ -65,9 +65,6 @@ func getBaseOutput(c configurationItem) (Output, error) {
 	if c.ResourceType == "" {
 		return Output{}, ErrMissingValue{Field: "ResourceType"}
 	}
```

Comment:
this is where I removed the check for configurationItem.ARN check; I know it would be ideal to check here if the ARN is retrievable from another location but the JSON is not yet unmarshalled into the full object until the code snippet in ec2.go